extensions/wikia/Search/classes/IndexService/DefaultContent.php

```diff
@@ -1,52 +1,59 @@
 <?php
 /**
  * Class definition for \Wikia\Search\IndexService\DefaultContent
+ *
  * @author relwell
  */
 namespace Wikia\Search\IndexService;
+
 use JsonFormatService;
 use Wikia\JsonFormat\JsonFormatSimplifier;
 use Wikia\Search\Utilities, simple_html_dom;
+
 /**
  * This is intended to provide core article content
+ *
  * @author relwell
  * @package Search
  * @subpackage IndexService
  */
-class DefaultContent extends AbstractService
-{
-	const SPACE_SEQUENCE_REGEXP = "/\s+/";  
-	
+class DefaultContent extends AbstractService {
+	const SPACE_SEQUENCE_REGEXP = "/\s+/";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "/\s+/" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)